src/Routing/Route/DashedRoute.php

```diff
@@ -56,11 +56,12 @@ protected function _camelizePlugin($plugin)
      * camelBacked form.
      *
      * @param string $url The URL to parse
+     * @param string $method The HTTP method.
      * @return array|false An array of request parameters, or false on failure.
      */
-    public function parse($url)
+    public function parse($url, $method = '')
```

Comment:
@chinpei215  The

> Add an argument to an overridden method　No

Line is from the 'Using CakePHP' section which explains what kind of modifications/overrides people can do to CakePHP code in a backwards compatible way. That line is no, because under 'Working on CakePHP'

> Add argument with default value 	Yes

is present. Both the user and us cannot add arguments in a backwards compatible way. We (like symfony) have retained the ability to add arguments with default values.